### DISCUSSION TITLE
E-mail types

### DISCUSSION THREAD

**Discussion by TommsNL** [liedekef/events-made-easy#530] [Q&A] [answered]
Sorry if this question is answered before (couldn't find it in the docs).

When sending an Event related mail I see the options "Booking emails" ("Boeking mails") &  "Participant emails" ("Deelnemer mails")

Are the  "Participant emails" the people who are allowed to come to the event (excluding waiting list). And booking mails everyone (so the accepted participants including the waiting list) ?

**Comment by liedekef** [comment, marked as answer]
The bookings-type is a mail for each booking made (so it understands rsvp-related placeholders like the number of seats booked, the booking price, ...). The participants-type for everybody that made a booking (but not each booking individually), so it doesn't understand rsvp-related placeholders (but it understands people and event related placeholders).
